horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -75,6 +80,18 @@ public void testDataSetCreated(TestInfo info) throws InterruptedException {
       }
    }
 
+   @org.junit.jupiter.api.Test
+   public void testFindSchema(TestInfo info) throws JacksonException {
+      String raw = "{\"value\":42.0,\"$schema\":\"urn:io.hyperfoil.tools.horreum.svc.AlertingServiceTest:testNotifications(TestInfo):1.0\"}";
+      ObjectMapper mapper = new ObjectMapper();
+      JsonNode jsonNode = mapper.readTree(raw);
+      assertNotNull(jsonNode);
+      assertEquals(2, jsonNode.size());
+      assertFalse(jsonNode.path("$schema").isMissingNode());
+      String uri = jsonNode.path("$schema").asText();
+      assertEquals("urn:io.hyperfoil.tools.horreum.svc.AlertingServiceTest:testNotifications(TestInfo):1.0", uri);
+   }
+
```

Comment:
A test for the whole cycle of transformation, as well as edge-cases (no transformers, missing schema...) would be useful here as well.